### DISCUSSION TITLE
After installation, reboots into installation again

### DISCUSSION THREAD

**Discussion by genxfed** [Security-Onion-Solutions/securityonion#2294] [Unsupported Versions] [answered]
After installing 2.3.10 the system reboots and never displays a login. After rebooting it goes back to installing the ISO again. I've installed and tested this several times but each time I never get the login screen to proceed to the next stage of the installation.

**Comment by dougburks** [comment, marked as answer]
You may need to change the boot order of the system or remove the ISO.

**Reply by jsavary87** [reply]
Does this have to be installed with CentOS as the underlying OS? or can it be Ubuntu like in previous versions? The reason I ask is because VirtualBox doesnt have CentOS as an option.

**Reply by dougburks** [reply]
This discussion has already been marked as answered. Instead of replying to somebody else's discussion that has been marked as answered, please start a new discussion. Thanks!
